File: C# .NET 2019/EX11_LSS/NodeInput.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace EX11_LSS
{
    public partial class NodeInput : Form
    {
        int nodeID;

        public NodeInput()
        {
            InitializeComponent();
            ShowDialog();
        }

        private void SendIDButton_Click(object sender, EventArgs e)
        {
            nodeID = Convert.ToInt32(NodeIDTextBox.Text);
            Dispose();
        }

        public int NodeID
        {
            get { return nodeID; }
        }
    }
}

File: VB .NET 2019/EX11_LSS/NodeInput.vb
Public Class NodeInput
    Dim nodeIdentification As Integer
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        nodeIdentification = Convert.ToInt32(NodeIDTextBox.Text)
        Dispose()
    End Sub
    Public Function NodeID() As Integer
        Return nodeIdentification
    End Function
End Class